new-bot/src/bot/features/download-video.ts

```
@@ -348,37 +358,151 @@ composer.inlineQuery(/(https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA
+    //   if (!videoFilePath){
+    //     return;
+    //   }
+    //   const videoSendResult = await ctx.api.sendVideo(config.MEDIA_STORAGE_GROUP_ID, new InputFile(videoFilePath));
+    //   ctx.logger.debug(`Video send result: ${JSON.stringify(videoSendResult)}`);
+    //   return ctx.answerInlineQuery([InlineQueryResultBuilder.videoMp4("id-1", "Send Video", videoSendResult.video.file_id, "https://img.icons8.com/?size=512&id=eAMGjpJ4skFB&format=png", {caption: `<a href="${sourceUrl}">Original</a>`, parse_mode: "HTML", })]);
+    // }
+
+  } catch (error) {
+    ctx.logger.error(`Error processing inline query: ${error}`);
+  } finally {
+    return next();
```

Biome (1.9.4)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)